doc/sphinx/source/community/diagnostic.rst

```diff
@@ -81,3 +83,18 @@ statistics, domain, plottype, authors and references in the ``config-references.
   log_provenance(nc-file,plot_file,caption,statistics,domain,plottype,authors,references,input-files)
 
 Have a look at the example NCL diagnostic in ``esmvaltool/diag_scripts/examples/diagnostic.ncl`` for a complete example.
+
+Adding references
+=================
+Recipe and diagnostic script can include references.
+When ESMValCore (the ``esmvaltool`` command) runs a recipe, it will store citation information in `BibTeX <https://en.wikipedia.org/wiki/BibTeX>`__ format.
```

Comment:
This might sound confusing to the general users who install the core as external dependency without knowing much about it.
Suggestion: turn sentence to passive form.

```suggestion
When a recipe is run, citation information is stored in `BibTeX <https://en.wikipedia.org/wiki/BibTeX>`__ format.
```